src/ofxstatement/statement.py

```diff
@@ -51,11 +63,28 @@ class Statement(object):
 
     def __init__(self, bank_id=None, account_id=None,
                  currency=None, account_type="CHECKING"):
+        global _ids
+
         self.lines = []
         self.bank_id = bank_id
         self.account_id = account_id
         self.currency = currency
         self.account_type = account_type
+        # See note generation of an id above
+        _ids = {}
+
+    def recalculate_balance(self):
+        recalculate_balance(self)
```

Comment:
Can you give an idea why/when this would be useful?

Reply:
Instead of importing a function that works on a class, I thought it would be more appropiate / OO style to define a method in that class. I just left the original function for backwards compability.

Reply:
Well, unless I'm missing something, balance recalculation should not depend on plugin, or specific subclass of `Statement`. Making it a method suggests that some plugin might have a different way of calculating the sum of all statement lines, and I cannot find a reason when this could be useful.

So, dunno, if you had some use for the method in mind, let's find a good explaination and put it to docstring. If the method is solely for "completeness", let's not do it right now.